main.js

```diff
@@ -1 +1,14 @@
-console.log("HELLO")
\ No newline at end of file
+const textEditor = document.querySelector(".text-editor")
+const preview = document.querySelector(".preview")
+var showdown = require('showdown')
```

Comment:
Why `var` here instead of `const`?

Reply:
There is no technical reason why I used **var** in this case as it's being defined globally, I should have used **const** for the sake of consistency.
The mistake i have done was to copy the code from the Package documentation.